sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/core/AbstractDownloadTest.java

```diff
@@ -17,7 +19,8 @@ public AbstractDownloadTest(TOptions options) {
     // Upload one blob for the whole test run. All tests can download the same blob
     public Mono<Void> globalSetupAsync() {
         return super.globalSetupAsync()
-            .then(blobAsyncClient.upload(createRandomByteBufferFlux(options.getSize()), null))
+            .then(blobAsyncClient.upload(createRandomByteBufferFlux(options.getSize()), new ParallelTransferOptions()
```

Comment:
Can you add a comment explaining why we are overriding the default upload size and block size?  Is 1GB the max, or could we set it even higher (in case we want to start testing blobs > 1GB)?

Reply:
Left a comment explaining this